pygmt/datasets/earth_magnetic_anomaly.py

```diff
@@ -46,6 +51,13 @@ def load_earth_magnetic_anomaly(resolution="01d", region=None, registration=None
         a pixel-registered grid is returned unless only the
         gridline-registered grid is available.
 
+    mag4km : bool
+        Choose the data version to use. The default is ``False``, which is
+        observed at sea level over oceanic regions and have no data over land.
+        Setting ``mag4km`` to ``True`` to use a version where all observations
+        are relative to an altitude of 4 km above the geoid and includes data
+        over land.
```

Comment:
I am not totally sure here, but maybe consistently singular or plural within the sentences sounds better?
```suggestion
    mag4km : bool
        Choose the data version to use. The default is ``False``, which is
        observed at sea level over oceanic regions and has no data over land.
        Set ``mag4km`` to ``True`` to use a version where all observations
        are relative to an altitude of 4 km above the geoid and include data
        over land.
```